src/aliceVision/hdr/hdr_test.cpp

```diff
@@ -78,7 +78,7 @@ bool buildBrackets(std::vector<std::string>& paths, std::vector<float>& times, c
 
         ALICEVISION_LOG_INFO("writing to " << temp.string());
 
-        image::writeImage(temp.string(), img_bracket, image::EImageColorSpace::LINEAR);
+        image::writeImage(temp.string(), img_bracket, image::EImageColorSpace::SRGB_LINEAR);
```

Comment:
This will break hdr_test.cpp (which is currently disabled, but we shouldn't introduce unnecessary changes that break existing code).

Reply:
This change makes matching AliceVision color space naming with OCIO one. In OCIO "linear" points out the color space defined to be the scene linear one. It could be ACEScg, Rec709 linear, or any other linear color space. This change allows to take advantage on that in AV.

Reply:
@demoulinv I understand the intention, but this PR doesn't actually introduce SRGB_LINEAR. It introduced it in the first commit [[Add AliceVision Color Space metadata]](https://github.com/alicevision/AliceVision/pull/1170/commits/c1c77210b695536ed941af543cb323800ebb8f9a), but the second commit [SRGB_LINEAR becomes LINEAR to be more compliant with the OCIO config](https://github.com/alicevision/AliceVision/pull/1170/commits/eeae5f871dc0d009318b7952053f3c5c476d5924) reverted everything back.